src/ddmd/aggregate.d

```diff
@@ -64,8 +64,6 @@ extern (C++) abstract class AggregateDeclaration : ScopeDsymbol
     uint structsize;        // size of struct
     uint alignsize;         // size of struct for alignment purposes
     VarDeclarations fields; // VarDeclaration fields
-    Sizeok sizeok;          // set when structsize contains valid data
```

Comment:
Don't forget to update c++ headers for all field and function signature changes once you are done.